src/binary-reversal/index.js

```diff
@@ -3,6 +3,28 @@
  *
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+function binaryReversal(value) {
+  if (typeof value !== "string" || value instanceof String || !value) {
```

Comment:
`value instanceOf String` is a wrong condition here. Think you mean, `not String`